labtool2.0/src/components/pages/CoursePage.js

```diff
@@ -646,6 +649,15 @@ export const CoursePage = props => {
         <br />
         <br />
         <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
```

Comment:
I think we should use margin instead of multiple line break tags

Reply:
Possibly, or a spacer element